StudentOutreach/Models/ViewModel.swift

```diff
@@ -168,6 +168,7 @@ struct StudentAssignmentInfo: Hashable {
         let comps = formatter.personNameComponents(from: name)
         return comps?.givenName ?? name
     }
+    let lastCourseActivityAt: Date?
```

Comment:
Let's move this up so it's above `firstName`. I generally like to keep computed properties separate from static properties.

Reply:
Moved it up for consistency